.gitignore

```diff
@@ -12,6 +12,7 @@ example/generated/**
 .idea
 
 service/artifacts/*
+!service/artifacts/Config.toml
```

Comment:
why would Config.toml be in `service/artifacts`?

Reply:
Because it's like that in `spells` project, so it's a unified approach. It also allows to run the service from the artifacts directory using `mrepl` at the same time.

Reply:
I strongly believe that it's not a final solution. I think we need fcli to be able to provide a package with all services, spells and `Config.toml`s in a single directory/archive, so we could use it for distribution.